chromadb/utils/embedding_functions.py

```diff
@@ -235,6 +235,61 @@ def __call__(self, input: Documents) -> Embeddings:
         ]
 
 
+class VoyageAIEmbeddingFunction(EmbeddingFunction[Documents]):
+    def __init__(
+            self,
+            api_key: str,
+            model_name: str,
+            embed_batch_size: Optional[int] = None,
+            truncation: Optional[bool] = None,
+            show_progress_bar: Optional[bool] = False,
+    ):
+        try:
+            import voyageai
+        except ImportError:
+            raise ValueError(
+                "The cohere python package is not installed. Please install it with `pip install -U voyageai`"
```

Comment:
"The cohere python package is not installed.
->
"The voyage python package is not installed.